Source/BestProjectEver/Private/AI/BPE_AIController.cpp

```diff
@@ -28,6 +30,15 @@ void ABPE_AIController::BeginPlay()
 	InitializeReferences();
 }
 
+//----------------------------------------------------------------------------------------------------------------------
+void ABPE_AIController::CheckMatchState(const FName MatchState)
```

Comment:
I'm not a fan of functions that begin with `Check`. That doesn't tell me anything about the function. If it's supposed to be a callback for when the match state changes, it could say so, something like `OnMatchStateChanged`.

Reply:
![image](https://user-images.githubusercontent.com/85327912/173869605-ae914dae-62df-4f7d-a488-36255dafd651.png)

Reply:
![image](https://user-images.githubusercontent.com/85327912/173869743-13844ba3-59cf-4ccb-a9b6-b3755185b728.png)